blockchain/blockchain-wallet-v4-frontend
# feat(buy): implemented logic to get in consideration proper limits based on product

## Description (optional)
This PR add trading limits endpoint in use for proper finding what is min/max

## Testing Steps (optional)
Login to wallet try to buy DOT and enter 1, then click on buy min and try to buy, you should be able to buy it
